models/evc.py

```diff
@@ -1029,6 +1030,16 @@ def get_id_from_cookie(cookie):
         evc_id = cookie - (settings.COOKIE_PREFIX << 56)
         return f"{evc_id:x}".zfill(14)
 
+    def set_flow_table_grou_id(self, flow_mod: dict, vlan) -> dict:
```

Comment:
There's a typo in the name of this function, you probably meant `group` instead of `grou`.

Reply:
Corrected